### PR TITLE
Fix the warning when you create an externalized service

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

**Comment by brendandburns** [comment]
Closes #9566

CLA for gongysh2004 verified in #9567

File: pkg/kubectl/cmd/create.go [hunk 1]
```diff
@@ -126,8 +126,8 @@ func printObjectSpecificMessage(obj runtime.Object, out io.Writer) {
 			An external load-balanced service was created.  On many platforms (e.g. Google Compute Engine),
```
Comment (thockin) [inline pkg/kubectl/cmd/create.go]: An ugly side effect of using back-quotes here is that all the whitespace at the front of lines is preserved.  Can we undent that?

**Comment by thockin** [comment]
LGTM

**Comment by thockin** [comment]
already lgtm

**Comment by jlowdermilk** [comment]
@k8s-bot please test this

**Comment by alex-mohr** [comment]
ok to test

**Comment by alex-mohr** [comment]
LGTM

**Pull request merged**